keyboards/wheatfield/blocked65/blocked65.c

```diff
@@ -0,0 +1,13 @@
+#include "blocked65.h"
+
+void matrix_init_kb(void) {
+	// put your keyboard start-up code here
+	// runs once when the firmware starts up
+	matrix_init_user();
+};
+
+void matrix_scan_kb(void) {
+	// put your looping keyboard code here
+	// runs every cycle (a lot)
+	matrix_scan_user();
+};
```

Comment:
Both these functions can be deleted here; they're the same as the default defined in QMK's core.

https://github.com/qmk/qmk_firmware/blob/4b7a3b2265d476dc3105511c74e25adc13ab8a8a/quantum/matrix_common.c#L21-L23

```suggestion
```